Tools/MonoGame.Content.Builder.Task/MonoGame.Content.Builder.Task.props

```diff
@@ -3,7 +3,11 @@
   <PropertyGroup>
     <DotnetCommand Condition="'$(DotnetCommand)' == ''">dotnet</DotnetCommand>
     <EnableMGCBItems Condition="'$(EnableMGCBItems)' == ''">true</EnableMGCBItems>
+    <!-- Allow users the ability to disble tool restoration if needed -->
+    <AutoRestoreMGCBTool Condition="'$(AutoRestoreMGCBTool)' == ''">true</AutoRestoreMGCBTool>
+    <MGCBToolDirectory>$(MSBuildThisFileDirectory)dotnet-tools/</MGCBToolDirectory>
     <MGCBCommand Condition="'$(MGCBCommand)' == ''">mgcb</MGCBCommand>
+    <MonoGameVersion>3.8.1.1-develop</MonoGameVersion>
```

Comment:
Do we need a better name for this? It will be replaced with the actual version of the app at build time on CI.

Reply:
You forgot to check the Condition before setting the variable.

Reply:
I don't think the user should be overriding this. This will be the version of the NuGet package these targets were shipped in. Unless we want users to install the targets from one version of Mg but install the tooling from another?

Reply:
I added the check anyway.